frontend/shared/src/constants.ts

```diff
@@ -10,3 +10,11 @@ export const POSTAL_CODE_REGEX = /^\d{5}$/;
 export const NAMES_REGEX =
   /^[\w',.ÄÅÖäåö-][^\d!#$%&()*+/:;<=>?@[\\\]_{|}~¡¿÷ˆ]+$/;
 export const CITY_REGEX = /^[ A-Za-zÄÅÖäåö-]+$/;
+
+export const EMAIL_REGEX =
+  // eslint-disable-next-line security/detect-unsafe-regex
+  /^(([^\s"(),.:;<>@[\\\]]+(\.[^\s"(),.:;<>@[\\\]]+)*)|(".+"))@((\[(?:\d{1,3}\.){3}\d{1,3}])|(([\dA-Za-z-]+\.)+[A-Za-z]{2,}))$/;
```

Comment:
The e-mail regex on [this list from OWASP](https://owasp.org/www-community/OWASP_Validation_Regex_Repository) might be useful if you don't want to disable the safety rule.

Reply:
Funny enough it still complains about the security but also about unicorn/better-regex. I took the advice from unicorn and it turn out to this: 
```
  // eslint-disable-next-line security/detect-unsafe-regex
  /^[\w&*+-]+(?:\.[\w&*+-]+)*@(?:[\dA-Za-z-]+\.)+[A-Za-z]{2,7}$/
```
It's shorter and probably safer though if it's from owasp.